### ISSUE TITLE
[bug]: Lora prompt causes index out of range error

### ISSUE THREAD

**Issue by slavikshen** [invoke-ai/InvokeAI#3157]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

macOS

### GPU

mps

### What version did you experience this issue on?

2.3.4

### What happened?

After I select the lora in the UI, the withLora(Shenhe_Hard,0.75) is appended to the end of the prompt.
However, the text gen crash for `list index out of range` error

The prompt is 
```
masterpiece (ultra-detailed) illustration (mature_female)++ 1girl stands under sakura tree blossom,(school_uniform,bustier).blend(1,1)  (mini frilled skirt) underboob++++ (detailed-embroidery) (full body) (detailed face) (\:3)++++ (seductive smile)   (hair_ornament) (detailed eyes) (falling_petals)++:3 

(detailed background) (insanely detailed and intricate) (hyper maximalist) (light from above):1 

withLora(Shenhe_Hard,0.75) 
```

The traceback is 
```
Traceback (most recent call last):
  File "/Volumes/Work/InvokeAI/.venv/lib/python3.10/site-packages/invokeai/backend/invoke_ai_web_server.py", line 1353, in generate_images
    self.generate.prompt2image(
  File "/Volumes/Work/InvokeAI/.venv/lib/python3.10/site-packages/ldm/generate.py", line 525, in prompt2image
    uc, c, extra_conditioning_info = get_uc_and_c_and_ec(
  File "/Volumes/Work/InvokeAI/.venv/lib/python3.10/site-packages/ldm/invoke/conditioning.py", line 58, in get_uc_and_c_and_ec
    legacy_blend = try_parse_legacy_blend(positive_prompt_string, skip_normalize_legacy_blend)
  File "/Volumes/Work/InvokeAI/.venv/lib/python3.10/site-packages/ldm/invoke/conditioning.py", line 229, in try_parse_legacy_blend
    flattened_prompts = [x.prompts[0] for x in parsed_conjunctions]
  File "/Volumes/Work/InvokeAI/.venv/lib/python3.10/site-packages/ldm/invoke/conditioning.py", line 229, in <listcomp>
    flattened_prompts = [x.prompts[0] for x in parsed_conjunctions]
IndexError: list index out of range
```

**Comment by slavikshen** [comment]
The workaround is to move the withLora from the end to the beginning.
```
withLora(Shenhe_Hard,0.75) masterpiece (ultra-detailed) illustration (mature_female)++ 1girl stands under sakura tree blossom,(school_uniform,bustier).blend(1,1)  (mini frilled skirt) underboob++++ (detailed-embroidery) (full body) (detailed face) (\:3)++++ (seductive smile)   (hair_ornament) (detailed eyes) (falling_petals)++:3 

(detailed background) (insanely detailed and intricate) (hyper maximalist) (light from above):1 
```

**Comment by lstein** [comment]
@damian0815 It appears that when a withLora() fragment appears at the end of a long prompt you get the indicated crash. I believe this may have to do with exceeding the token limit. Could you take a look?

**Comment by lstein** [comment]
Confirmed this bug fixed in the latest compel library, 1.1.3.
